### PR TITLE
Initial board render

### PR DESCRIPTION
**Pull request by s373r**

### CODE REVIEW

File: src/renderer.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import curses.textpad
+
+from board import Board
+
+
+class Renderer:
+    __BORDER_WEIGHT = 1
```
Comment (alena-bartosh) [inline src/renderer.py]:
```suggestion
    __BORDER_WEIGHT: int = 1
```

File: src/renderer.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        board_cells = board.get_cells()
+
+        for x in range(board.height):
+            for y in range(board.weight):
+                is_cell_fill = board_cells[x][y]
+
+                self.__draw_cell(x, y, is_cell_fill)
+
+    # TODO(DP): use Figure class instead of figure_points
+    def __draw_figure(self, figure_points: list):
```
Comment (alena-bartosh) [inline src/renderer.py]:
```suggestion
    def __draw_figure(self, figure_points: list) -> None:
```

File: src/renderer.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        for cell in figure_points:
```
Comment (alena-bartosh) [inline src/renderer.py]:
```suggestion
        for point in figure_points:
```

Comment (alena-bartosh) [reply src/renderer.py]: We draw cell by points, so propose to use "point"

**Review by alena-bartosh — COMMENTED** [review]
`def render()` is done in very nice way

File: src/base_engine_event_handler.py [hunk 4]
```diff
@@ -1,12 +1,14 @@
 from abc import ABC, abstractmethod
 
+from board import Board
 from engine_event import EngineEvent
 from game_event import GameEvent
 
 
 class BaseEngineEventHandler(ABC):
-    def __init__(self, stdscr):
+    def __init__(self, stdscr, board: Board):
```
Comment (alena-bartosh) [inline src/base_engine_event_handler.py]:
```suggestion
    def __init__(self, stdscr, board: Board) -> None:
```

File: src/engine_event_handler.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+emulated_figure_cells = [
+    Point(0, 5),
+    Point(1, 6),
+    Point(1, 5),
+    Point(2, 6),
+]
+
+
+class EngineEventHandler(BaseEngineEventHandler):
+    def __init__(self, stdscr, board: Board):
```
Comment (alena-bartosh) [inline src/engine_event_handler.py]:
```suggestion
    def __init__(self, stdscr, board: Board) -> None:
```

File: src/renderer.py (hunk 1 above)
Comment (s373r) [reply src/renderer.py]: Done

File: src/renderer.py (hunk 2 above)
Comment (s373r) [reply src/renderer.py]: Done

File: src/renderer.py (hunk 3 above)
Comment (s373r) [reply src/renderer.py]: Done

File: src/base_engine_event_handler.py (hunk 4 above)
Comment (s373r) [reply src/base_engine_event_handler.py]: Done

**Review by alena-bartosh — DISMISSED** [review]

**Review by alena-bartosh — APPROVED** [review]

**Review by Serg-Bartosh — APPROVED** [review]
Well done :grinning: :+1:

**Pull request merged**